lms/djangoapps/ccx/views.py

```diff
@@ -554,8 +556,22 @@ def ccx_gradebook(request, course, ccx=None):
             for student in enrolled_students
         ]
 
+        max_students_per_page = 20
```

Comment:
In my opinion this should go in a global setting or something like that, but I'm not sure how the rest of the code is organized, so maybe just wait for Sarina's input.